src/main/java/seedu/address/model/salesorder/SalesOrder.java

```diff
@@ -0,0 +1,64 @@
+package seedu.address.model.salesorder;
+
+import seedu.address.model.task.Date;
+
+public class SalesOrder {
+    private static int count = 1;
```

Comment:
This might not work properly for sales id generation because the count is reset everytime we restart the application. 
I guess I'll change this part myself when I implement the addorder command.